pkg/apis/planetscale/v2/vitesscluster_types.go

```diff
@@ -107,6 +107,37 @@ type VitessClusterSpec struct {
 	// not have any prefix (just "flag", not "-flag"). To set a boolean flag,
 	// set the string value to either "true" or "false".
 	ExtraVitessFlags map[string]string `json:"extraVitessFlags,omitempty"`
+
+	// TopoReconciliation can be used to enable or disable registration or pruning of various vitess components to and from topo records.
+	TopoReconciliation TopoReconcileConfig `json:"topoReconciliation,omitempty"`
+}
+
+// TopoReconcileConfig can be used to turn on or off registration or pruning of specific vitess components from topo records.
+// This should only be necessary if you need to override defaults, and shouldn't be required for the vast majority of use cases.
+type TopoReconcileConfig struct {
+	// RegisterCellsAliases can be used to enable or disable registering cells aliases into topo records.
```

Comment:
Add an extra line to each of these doc comments to give the default value in the format we use throughout the rest of the API:

```go
// Default: true
```

Reply:
Done https://github.com/planetscale/vitess-operator/pull/26/commits/a472f7bf82e16589110301ada180af75a5b98930.